### PR TITLE
Add ability to mount nfs shares

### PR DESCRIPTION
**Pull request by roedie**
Since this is used to install the NFS server, client and manage
the exports file I think it should also manage the NFS mounts

### CODE REVIEW

File: nfs/mount.sls [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+include:
+  - nfs.client
+
+{% for m in salt['pillar.get']('nfs:mount').iteritems() %}
+{{ m[1].mountpoint }}:
+  mount.mounted:
+    - device: {{ m[1].location }}
+    - fstype: nfs
+    - opts: {{ m[1].opts|default('vers=3') }}
+    - persist: {{ m[1].persist|default('True') }}
```
Comment (iggy) [inline nfs/mount.sls:12]: you don't have an example of persist in pillar.example

File: nfs/mount.sls [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    - mkmnt: {{ m[1].mkmnt|default('True') }}
```
Comment (iggy) [inline nfs/mount.sls:13]: same... not deal breakers, just saying

Comment (roedie) [inline nfs/mount.sls:13]: Added both persist and mkmnt.

**Comment by iggy** [comment]
Thanks a lot man :)

**Pull request merged**